### ISSUE TITLE
Audio Device Not Detected

### ISSUE THREAD

**Issue by mircoianese** [gnodipac886/MatebookXPro-hackintosh#105]
Hello! First of all thanks for the amazing work done here.

I installed OSX 10.15.4 on my 2018 Matebook X Pro. Got almost everything working except for audio. 

Under `Settings - Sound` there are no output / input devices. From IORegistryExplorer I see that AppleALC is loading correctly.

I also tried to change layout-ids in Clover but with no luck. 
I have not changed anything from the default config.plist provided in this repo. 

Any idea?
Thank you

**Comment by Chatbox1024** [comment]
AppleALC.kext v1.4.8 doesn't work.  The latest one that works for me is v1.4.7

**Comment by mircoianese** [comment]
> AppleALC.kext v1.4.8 doesn't work. The latest one that works for me is v1.4.7

Still no luck. Replaced AppleALC-m8.kext with the 1.4.7 AppleALC.kext.
Still no audio device

**Comment by Chatbox1024** [comment]
What have / haven't you tried?

**Comment by mircoianese** [comment]
> What have / haven't you tried?

This is what I tried so far:

-Checked that the Device Path is correct (and it is):
<img width="862" alt="image" src="https://user-images.githubusercontent.com/22716366/80302339-b68ae480-87a9-11ea-817c-df2e0a8eac74.png">

-Tried to change Apple HDA kext in Clover/kexts/other

-Made sure that the Kext is loading, and it seems to:
<img width="736" alt="image" src="https://user-images.githubusercontent.com/22716366/80302366-ed60fa80-87a9-11ea-854b-cba35ccd022f.png">

-Tried different layout-ids (5, 11, 13, 14, 21, 22, 28, 56, 57, 66, 97)

And the result is that no sound card is detected at all (so I'm not totally sure this is a layout-id issue):
<img width="743" alt="image" src="https://user-images.githubusercontent.com/22716366/80302396-141f3100-87aa-11ea-8912-00d585b1232e.png">

Another thing I noticed is that Clover is adding at boot params that I did not specified in the config (for example -v). If I try to delete that param by pressing "O" at boot, it still boots in verbose mode. So I'm not totally sure if this can be related.
I will also attach my Clover folder.

[CLOVER.zip](https://github.com/gnodipac886/MatebookXPro-hackintosh/files/4534931/CLOVER.zip)

Thank you for your help
